### PR TITLE
Example drbd test

### PR DESCRIPTION
**Pull request by mmichal10**

### CODE REVIEW

**Comment by mmichal10** [comment]
Requires https://github.com/intel-innersource/frameworks.validation.opencas.test-framework/pull/34/files

**Comment by bkarpins** [comment]
retest this please

**Review by karolinavelkaja — DISMISSED** [review]

File: test/functional/tests/example/example_test.py [hunk 1]
```diff
@@ -147,3 +151,71 @@ def test_example_multidut():
...
+            drbd_dev = TestRun.disks["drbd_device"]
+            drbd_md_dev = RamDisk.create(Size(100, Unit.MebiByte), 1)[0]
+            drbd_dev.create_partitions([Size(200, Unit.MebiByte)])
+            drbd_dev = drbd_dev.partitions[0]
+
+            nodes.append(
+                Node(TestRun.dut.hostname, drbd_dev.path, drbd_md_dev.path, dut2.ip, "7790")
+            )
+        drbd_resurce_name = "caches"
+        caches = Resource(drbd_resurce_name, "/dev/drbd0", nodes)
```
Comment (pdebski21) [inline test/functional/tests/example/example_test.py:194]: If we use drbd_resurce_name helper variable maybe adding drbd_exported_object as variable will be plus.

File: test/functional/tests/example/example_test.py [hunk 2]
```diff
@@ -147,3 +151,71 @@ def test_example_multidut():
...
+    with TestRun.use_dut(dut1), TestRun.step(f"Set {dut1} as primary node"):
+        primary.set_primary(force=True)
+
+    with TestRun.use_dut(dut1), TestRun.step("Wait for drbd to sync"):
+        primary.wait_for_sync()
+
+    with TestRun.step("Test cleanup"):
+        for dut in TestRun.duts:
+            with TestRun.use_dut(dut):
+                Drbdadm.down(caches.name)
```
Comment (pdebski21) [inline test/functional/tests/example/example_test.py:221]: starting cache will be additional value to check if drbd clean up works fine

File: test/functional/tests/example/example_test.py [hunk 3]
```diff
@@ -147,3 +151,71 @@ def test_example_multidut():
...
+                Node(TestRun.dut.hostname, drbd_dev.path, drbd_md_dev.path, dut1.ip, "7790")
+            )
+        with TestRun.use_dut(dut2):
+            TestRun.dut.hostname = TestRun.executor.run_expect_success("uname -n").stdout
+            TestRun.LOGGER.info(TestRun.dut.hostname)
+
+            drbd_dev = TestRun.disks["drbd_device"]
+            drbd_md_dev = RamDisk.create(Size(100, Unit.MebiByte), 1)[0]
+            drbd_dev.create_partitions([Size(200, Unit.MebiByte)])
+            drbd_dev = drbd_dev.partitions[0]
```
Comment (pdebski21) [inline test/functional/tests/example/example_test.py:188]: we can use `for dut in TestRun.duts:` here

File: test/functional/tests/example/example_test.py (hunk 1 above)
Comment (pdebski21) [reply test/functional/tests/example/example_test.py]: One way or another 😅

**Review by karolinavelkaja — APPROVED** [review]

**Review by KlaudiaJ — APPROVED** [review]

**Review by karolinavelkaja — APPROVED** [review]

**Pull request merged**